Dockerfile

```diff
@@ -9,22 +9,14 @@ FROM --platform=$BUILDPLATFORM tonistiigi/xx:${XX_VERSION} AS xx
 
 FROM --platform=$BUILDPLATFORM rust:${RUST_VERSION} AS base
 COPY --from=xx / /
-RUN apt-get update -y && apt-get install --no-install-recommends -y clang jq
+RUN apt-get update -y && apt-get install --no-install-recommends -y clang jq wget
```

Comment:
Doesn't seem like we should need wget?

Reply:
`wasmedge-sys` needs `wget` to download the library.
We could fix that upstream for the next release of `wasmedge-sdk` and remove it then.
https://github.com/WasmEdge/wasmedge-rust-sdk/issues/30